### DISCUSSION TITLE
Enterprise Edition

### DISCUSSION THREAD

**Discussion by markgarrigan** [upptime/upptime#842] [Q&A] [answered]
We use Github Enterprise Cloud. One of the restrictions of our enterprise is that we can not use actions that are not approved. So currently upptime doesn't work because the following actions are not allowed...

upptime/uptime-monitor
benc-uk/workfow-dispatch
peaceiris/actions-gh-pages

We can use reusable workflows within the same org. So I could fork these in. However, I'm assuming those are actions are not currently configurable? I could update the workflow files to use our custom versions, but that change would probably be over written after Update Template runs?

**Comment by AnandChowdhary** [comment, marked as answer]
You're absolutely right about everything. I'd be happy to approve a PR that allows people to overwrite these for a custom action name! :)
